transaction.js

```diff
@@ -38,6 +38,7 @@ class Transaction {
             tempfail: 0,
             reject: 0,
         }
+        this.msg_status = 'undefined';
```

Comment:
'undefined' shouldn't be quoted. It should be one of: `undefined`, `null`, or an empty string `''`.

Reply:
The reason for a string is because faulty Kibana/ES didn't know how to output "missing" count aggregate. They've added that in https://www.elastic.co/blog/kibana-6-2-0-released , so we should be golden. We could also pre-process that in ES plugin, substituting missing/undefined value with a string.